docs/crio.conf.5.md

```diff
@@ -25,143 +23,188 @@ No bare options are used. The format of TOML can be simplified to:
     option = value
 
 ## CRIO TABLE
+CRI-O reads its storage defaults from the containers-storage.conf(5) file located at /etc/containers/storage.conf. Modify this storage configuration if you want to change the system's defaults. If you want to modify just CRI-O, you can change the storage configuration options here.
```

Comment:
suggest: "modify just CRI-O" to "modify storage just for CRI-O"

Reply:
I agree.

Reply:
I opened https://github.com/containers/storage/issues/209 for that and optimistically made the that statement already here :-D